offices/_single.html

```diff
@@ -1,8 +1,11 @@
 {% extends "layout-side-nav.html" %}
 {% import "_vars-offices.html" as vars with context %}
 
+{% set office_title = office.title | safe if office.title %}
```

Comment:
We were using `office.title` numerous times in the template so I just created a variable. Looks like its going to change based on team review though. I will check if an else condition is needed.